resources/testdata/glyphs3/WghtVar.glyphs

```diff
@@ -313,20 +313,20 @@ unicode = 61;
 );
 kerningLTR = {
 m01 = {
+"@MMK_L_bracketleft_R" = {
+exclam = -165;
+};
 bracketleft = {
 bracketright = -300;
 };
 exclam = {
-"@MMK_R_brackets" = -160;
+"@MMK_R_bracketright_L" = -160;
```

Comment:
this kern pair is lost (intentionally?) because you haven't renamed the bracketright glyph kerning groups accordingly, they are still called "brackets" for both L and R

Reply:
Not intentionally! Fixed.